pkg/synthesis/model.go

```diff
@@ -117,19 +117,25 @@ func strAdminPolicy(policy *symbolicPolicy, options *SynthesisOptions) string {
 
 func strGroups(allGroups []*collector.Group, color bool) string {
 	// todo: identify here cases in which we were unable to process expr
-	header := []string{"Group", "Expression", "VM"}
+	header := []string{"Group", "Expression", "VM", "Addresses", "Segment", "Transport Node", "IP Group"}
```

Comment:
there is some kind of duplication where groups details are printed here and in analysis log?

Reply:
updated so that group str func is originated from config str functionality, to be reused in both config str for analysis, and for abstract model print.